registration/tests.py

```diff
@@ -481,6 +481,10 @@ def test_create_registration_fires_tasks(self):
                       "http://test/v2/json/subscription",
                       body='jembi_post_json task', status=201,
                       content_type='application/json')
+        responses.add(responses.POST,
+                      "http://test/v2/registration/net.ihe/DocumentDossier",
+                      body='jembi_post_xml task', status=201,
```

Comment:
Proper response here `Request added to queue` is good, everything else is bad. Also it's 202's from Jembi.